plugins/org.locationtech.udig.project.edit/src/org/locationtech/udig/project/internal/provider/ProjectItemProviderAdapterFactory.java

```diff
@@ -498,6 +530,12 @@ public void dispose() {
             contextModelItemProvider.dispose();
         if (editManagerItemProvider != null)
             editManagerItemProvider.dispose();
+        if (layerItemProvider != null)
+            layerItemProvider.dispose();
+        if (mapItemProvider != null)
+            mapItemProvider.dispose();
```

Comment:
So I guess its the minimal impact which keeps the member mapItemProvider (which is unused). fine for me